submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/itest/PropertyHolder.kt

```diff
@@ -0,0 +1,44 @@
+package ac.uk.ebi.biostd.itest.itest
+
+import org.yaml.snakeyaml.DumperOptions
+import org.yaml.snakeyaml.Yaml
+import java.io.File
+
+/**
+ * Holds application properties and write them into its YAML representation.
+ */
+class PropertyHolder {
```

Comment:
I'd suggest `ConfigurationPropertiesHolder`, `ConfigurationPropertiesWriter` or `ConfigurationPropertiesHelper` which is a little bit more specific to what it actually does